lib/matplotlib/axes/_base.py

```diff
@@ -438,7 +438,8 @@ def __init__(self, fig, rect,
           ================   =========================================
           Keyword            Description
           ================   =========================================
-          *adjustable*       [ 'box' | 'datalim' | 'box-forced']
+          *adjustable*       [ 'box' | 'datalim' | 'xlim' | 'ylim' |
```

Comment:
this has to be one line or have a trailing `\` because we still have some home-grown docstring parsing.